src/rgw/setup/rgw.py

```diff
@@ -322,15 +334,36 @@ def _check_consul_connection(conf: MappedConf):
             raise SetupError(errno.EINVAL, f"Consul server {host:port} not reachable")
 
     @staticmethod
-    def _fetch_endpoint_url(conf: MappedConf, confstore_endpoint_key: str, endpoint_type: str):
+    def _get_gconf_key_list(conf: MappedConf, gconf_num_key:str, actual_gconf_key:str):
+        """Get value list of specified gconf key."""
```

Comment:
Am not able to visualize the key value(list) pairs here, can you add examples in code itself

Reply:
done

Reply:
Good examples